index.html

```diff
@@ -0,0 +1,183 @@
+        <section id="program">
+            <div class="program-title">
+                <h2>Why Kefi Unearthed?</h2>
+                <hr>
+            </div>
+            <div class="wrapper program-content">
+                <div class="program-card">
+                    <div class="card-icon">
+                        <img src="images/icons/icon1.png" alt="icon 1">
+                        <h3>Hope</h3>
+                    </div>
+                    <div class="card-description">
+                        <p>Now the God of hope fill you with all joy and peace in believing, that ye may abound in hope, through the power of the Holy Ghost.<br> - Romans 15:13</p>
+                    </div>
+                </div>
+                <div class="program-card">
+                    <div class="card-icon">
+                        <img src="images/icons/icon2.png" alt="icon 2">
+                        <h3>Faith</h3>
+                    </div>
+                    <div class="card-description">
+                        <p>If ye have faith as a grain of mustard seed, ye shall say unto this mountain, Remove hence to yonder place; and it shall remove; and nothing shall be impossible unto you. <br> - Matthew 17:20</p>
+                    </div>
+                </div>
+                <div class="program-card">
+                    <div class="card-icon">
+                        <img src="images/icons/icon3.png" alt="icon 3">
+                        <h3>Prayer</h3>
+                    </div>
+                    <div class="card-description">
+                        <p>The prayer of faith shall save the sick, and the Lord shall raise him up; and if he have committed sins, they shall be forgiven him. <br> - James 5:14-15</p>
+                    </div>
+                </div>
+                <div class="program-card">
+                    <div class="card-icon">
+                        <img src="images/icons/icon4.png" alt="icon 4">
+                        <h3>Heirs</h3>
+                    </div>
+                    <div class="card-description">
+                        <p>And if children, then heirs; heirs of God, and joint-heirs with Christ; if so be that we suffer with him, that we may be also glorified together. <br> - Romans 8:17</p>
+                    </div>
+                </div>
+                <div class="program-card">
+                    <div class="card-icon">
+                        <img src="images/icons/icon5.png" alt="icon 5">
+                        <h3>Confess</h3>
+                    </div>
+                    <div class="card-description">
+                        <p>Whosoever therefore shall confess me before men, him will I confess also before my Father which is in heaven. <br> - Matthew 10:32</p>
+                    </div>
+                </div>
+            </div>
+            <div class="program-btn">
+                <button type="button">Join us and Spread the Love of Jesus</button>
+            </div>
```

Comment:
Thank you for your comment. I ask that you please don't make me change my customized design. 🙏 I purposely want a button to expand like a banner. I want the button to remain red and I want it to expand across all five cards. The reason for the button to expand to all five cards is because five is a symbolic Biblical number, which means mercy. I would really appreciate you allowing me to keep my customized design. Thank you.